remappings.txt

```diff
@@ -6,3 +6,4 @@
 @vaults/=src/
 @tests/=lib/morpho-contracts/test-foundry/
 @contracts/=lib/morpho-contracts/contracts/
+@forge-std/=lib/morpho-contracts/lib/forge-std/src/
```

Comment:
You added forge-std as a dependency of this repo but you are not using it here. I suggest either use the morpho-v1's forge-std version (which is better because we need it to compile morpho-v1 contracts), or map the forge-std library referenced in this repo

Reply:
What you're doing here is mapping `@forge-std` to `morpho-v1`'s forge-std version. But in this PR, you also added `forge-std` to this repository. So the submodule added to this repository is currently unused. I suggest removing it

Reply:
Oh right, good catch !